spec/features/skills_form_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+
+describe 'Skill Form', type: :system, js:true do
+  before do
+    visit skills_path
+    click_link(href: new_skill_path)
+    @category = categories(:ruby)
+    @radar = "hold"
+    @portfolio = "passiv"
+  end
+
+  it 'creates the skill when the form is submitted' do
+    fill_in 'skill_title', with: 'New Skill Title'
+    select @category.title, from: 'skill_category_id'
+    select @radar, from: 'skill_radar'
+    select @portfolio, from: 'skill_portfolio'
+    click_button 'Skill erstellen'
+    all 'tbody tr', count: 6
```

Comment:
Seems like it is a little bit too unspecific. If we added a table somewhere else this test would pass for the wrong reason.
Maybe check for the 6 names of the skills instead?